Sources/Authgear.swift

```diff
@@ -1097,8 +1101,10 @@ public class Authgear {
                     wechatRedirectURI: wechatRedirectURI,
                     page: nil,
                     settingsAction: settingsAction,
-                    authenticationFlowGroup: nil
-                ), verifier: nil)
+                    authenticationFlowGroup: nil,
+                    responseMode: nil,
+                    xAppInitiatedSSOToWebToken: nil
+                ), clientID: self.clientId, verifier: nil)
```

Comment:
This line should conflict with Elise recent change.https://github.com/authgear/authgear-sdk-ios/pull/175

Question: Do we need to supply code verifier in pre-authenticated URL flow?

Reply:
No need, because it does not return a code.